webapp/src/components/legal_hold_table/legal_hold_table.tsx

```diff
@@ -22,19 +23,28 @@ const LegalHoldTable = (props: LegalHoldTableProps) => {
         ),
     );
 
+    const group_ids = Array.from(
+        new Set(
+            legalHolds.map((lh) => lh.group_ids).filter((i) => i !== null).reduce((prev, cur) => prev.concat(cur), []).filter((i) => i !== null),
```

Comment:
🤯

Reply:
This is hard to read, can we simplify it?

Reply:
We can't really simplify the work that's happening here, but I've added comments to explain what's happening. Is that good enough? Or do we want to re-write this in a more verbose way?